payments/voucher_manager.go

```diff
@@ -0,0 +1,136 @@
+package payments
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// paymentStatus stores the status of payments for a given payment channel.
+type paymentStatus struct {
+	channelSender   common.Address
```

Comment:
Nit: I would prefer payer / payee over sender / receiver?

Reply:
Done in https://github.com/statechannels/go-nitro/pull/844/commits/659ea22d1d8624973eb4e08cc0f41515b2744c78